sdks/js/src/types.ts

```diff
@@ -1584,6 +1585,30 @@ export type GetAgentConfigurationsResponseType = z.infer<
   typeof GetAgentConfigurationsResponseSchema
 >;
 
+export const CreateAgentConfigurationWithDefaultsRequestSchema = z.object({
```

Comment:
If we want to externally use the word "generic" on the public API we should try to match it on the SDK no?